sys/include/psa_crypto/psa/crypto_struct.h

```diff
@@ -28,34 +28,7 @@ extern "C" {
 #include "crypto_sizes.h"
 #include "crypto_contexts.h"
 
-/**
- * @brief   Structure containing a hash context and algorithm
- */
-struct psa_hash_operation_s {
-    psa_algorithm_t alg;        /**< Operation algorithm */
-#if IS_USED(MODULE_PSA_HASH)
-    psa_hash_context_t ctx;     /**< Operation hash context */
-#endif
-};
-
-/**
- * @brief   This macro returns a suitable initializer for a hash operation object of type
- *          @ref psa_hash_operation_t.
- */
-#define PSA_HASH_OPERATION_INIT { 0 }
-
-/**
- * @brief   Return an initial value for a hash operation object.
- *
- * @return  struct psa_hash_operation_s
- */
-static inline struct psa_hash_operation_s psa_hash_operation_init(void)
-{
-    const struct psa_hash_operation_s v = PSA_HASH_OPERATION_INIT;
-
-    return v;
-}
-
```

Comment:
Did you need to move this chunk below? Less changes that better (at least from the review perspective).

Reply:
I wanted to sort them by alphabet, but of course I can change this back

Reply:
I think it makes sense to maintain the new order to only have one continuous block of hash-related things to be guarded by the if-directive.